realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -16,19 +16,25 @@
 
 package io.realm;
 
-
 import android.annotation.SuppressLint;
 import android.os.Looper;
 
+import java.util.Date;
+import java.util.Locale;
+
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
+import io.realm.exceptions.RealmException;
 import io.realm.internal.CheckedRow;
 import io.realm.internal.Collection;
+import io.realm.internal.RealmObjectProxy;
 import io.realm.internal.Row;
 import io.realm.internal.SortDescriptor;
 import io.realm.internal.Table;
 import io.realm.internal.UncheckedRow;
+import io.realm.internal.Util;
+import io.realm.internal.android.JsonUtils;
```

Comment:
This looks like a mistake>

Reply:
Are we ok having android specific code in this class?

Reply:
Not really. It would have to be refactored for JVM support, but if no immediate replacement exist, we can just punt it to we actually need to